src/bin/log/tracing_chrome.rs

```diff
@@ -519,14 +520,17 @@ where
         }
     }
 
-    fn get_root_id(&self, span: SpanRef<S>) -> Option<u64> {
+    /// Returns `Option<i64>` instead of `Option<u64>` because apparently Perfetto gives an
+    /// error if an id does not fit in a 64-bit signed integer in 2's complement. We cast the
+    /// span id from `u64` to `i64` with wraparound, since negative values are fine.
```

Comment:
I don't think the reader has any reason to expect `u64` here... the comment is probably better placed near the cast.

Reply:
Done